From f2a1098630af846994788a7e8a00c152e463340c Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sun, 27 May 2018 01:03:37 +0300
Subject: [PATCH 3/3] Make wcnss_service build with the VNDK.

When the vndk is enabled (BOARD_VNDK_VERSION=current),
vendor libraries must only link against vendor variants
in the build system. This means that they receive restricted
sets of headers and that automatic global headers are restricted.

Test: build wcnss_service with BOARD_VNDK_VERSION=current
Change-Id: Idb0522aee5df0d190bedcec36055699c0af6034a
---
 wcnss-service/wcnss_service.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/wcnss-service/wcnss_service.c b/wcnss-service/wcnss_service.c
index f6f3a76..eafa81b 100644
--- a/wcnss-service/wcnss_service.c
+++ b/wcnss-service/wcnss_service.c
@@ -37,6 +37,7 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <utime.h>
 #include <sys/stat.h>
 #include <sys/sendfile.h>
+#include <unistd.h>
 #define LOG_TAG "wcnss_service"
 #include <cutils/log.h>
 #include <cutils/properties.h>
-- 
2.17.1

